### PR TITLE
[th]: Update php.json

### PR DESCRIPTION
**Pull request by bact**
- Make translations in `present`, `prohibited`, `required`, and `missing` set consistent in structure.
- Change "ฟิลด์" (field) to "ช่องข้อมูล" to make them the same translation as other instances of "field".
- Remove fullstops at the end of sentence, as they are not a symbol in standard Thai writing
- Remove few redundancies in translation
- Fix few typos, mistranslations, and spacing
- Change `:Attribute` to `:attribute`

### CODE REVIEW

File: locales/th/php.json [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+    "accepted_if": "ข้อมูล :attribute ต้องผ่านการยอมรับ เมื่อ :other เป็น :value",
     "active_url": "ข้อมูล :attribute ต้องเป็น URL เท่านั้น",
     "after": "ข้อมูล :attribute ต้องเป็นวันที่หลังจาก :date",
     "after_or_equal": "ข้อมูล :attribute ต้องเป็นวันที่ตั้งแต่วันที่ :date หรือหลังจากนั้น",
     "alpha": "ข้อมูล :attribute ต้องเป็นตัวอักษรภาษาอังกฤษเท่านั้น",
     "alpha_dash": "ข้อมูล :attribute ต้องเป็นตัวอักษรภาษาอังกฤษ ตัวเลข และ _ เท่านั้น",
     "alpha_num": "ข้อมูล :attribute ต้องเป็นตัวอักษรภาษาอังกฤษ ตัวเลข เท่านั้น",
     "array": "ข้อมูล :attribute ต้องเป็น array เท่านั้น",
-    "ascii": ":Attribute ต้องมีอักขระและสัญลักษณ์ที่เป็นตัวอักษรและตัวเลขคละกันแบบไบต์เดี่ยวเท่านั้น",
+    "ascii": ":attribute ต้องมีอักขระและสัญลักษณ์ที่เป็นตัวอักษรและตัวเลขคละกันแบบไบต์เดี่ยวเท่านั้น",
```
Comment (andrey-helldar) [inline locales/th/php.json:11]: Just a notification. [Read up](https://github.com/Laravel-Lang/common/discussions/7) on Laravel's internal mechanics if you haven't encountered them before.

Although, judging by the writing of the characters, in the Thai language letters are not divided into upper and lower case.

**Comment by andrey-helldar** [comment]
Trigger me when the PR is ready. Ok?

File: locales/th/php.json (hunk 1 above)
Comment (bact) [reply locales/th/php.json:11]: Thank you. I'm thinking about that as well. But all these strings in English they are `"The :attribute..."` and yes there's no concept of uppercase and lowercase in Thai script.

**Comment by bact** [comment]
@andrey-helldar it's ready now.

**Review by andrey-helldar — APPROVED** [review]
ขอบคุณ

**Pull request merged**